tests/showframes.lua

```diff
@@ -0,0 +1,13 @@
+return function()
+    local plain = require "classes.plain"
+    local class = plain()
+    SILE.documentState.documentClass = class
+
+    class:loadPackage("frametricks")
+
+    SILE.call("showframe",{ id = "all"})
+    for i = 1, 50 do
```

Comment:
When writing regression test, we want as little output as possible to prove the case. In this case just one supereject and a tiny bit of content should do the trick, no need for 50 pages. Unless you expect to stop functioning after passing 42 for ... galaxy reasons.